internal/webapi/server.go

```diff
@@ -109,6 +109,11 @@ func (a *Api) Authenticate(auth authentication.Authenticator[APIAuthCredientals]
 	// Check if credientals are correct
 	token, err := auth.CreateToken(packet)
 
+	if err == authentication.InvalidCredientalsError || err == authentication.NotAuthenticatedError {
```

Comment:
This should use [`errors.Is`](https://pkg.go.dev/errors#Is), to allow wrapped errors